.github/workflows/assets.yml

```diff
@@ -162,8 +162,8 @@ jobs:
         id: wheel
         working-directory: wheelhouse
         run: |
-          echo "WHEEL_NAME=$(ls *.whl)" >> "$GITHUB_OUTPUT"
```

Comment:
why remove the quotes now?

Reply:
Because that's something I had tried, to see if it was the source of the problem, but it wasn't. So I've preferred to leave them as they were before, since I think the code looks clearer.

Reply:
I heard it's actually a pretty good practice to quote variables in bash
Otherwise it might behave not like you expect when the variable is a string containing separators
But here we're safe